src/test/java/it/gov/pagopa/arc/fakers/bizEvents/paidnotice/BizEventsPaidNoticeDTOFaker.java

```diff
@@ -0,0 +1,33 @@
+package it.gov.pagopa.arc.fakers.bizEvents.paidnotice;
+
+import it.gov.pagopa.arc.connector.bizevents.dto.paidnotice.BizEventsPaidNoticeDTO;
+
+public class BizEventsPaidNoticeDTOFaker {
+    public static BizEventsPaidNoticeDTO mockInstance(Integer bias, boolean isCart){
+        return mockInstanceBuilder(bias, isCart).build();
+    }
+    public static BizEventsPaidNoticeDTO.BizEventsPaidNoticeDTOBuilder mockInstanceBuilder(Integer bias, boolean isCart) {
+        if (!isCart) {
```

Comment:
I would add a check to assert if all the fields are set.
This way you will verify that eventuallu new fields will be set...